COLLADAMaya/scripts/COLLADAMaya.vcxproj

```diff
@@ -3249,7 +3249,7 @@ copy $(TargetPath) ..\testsuite &gt; nul
     </ClCompile>
     <Link>
       <AdditionalOptions>/export:initializePlugin /export:uninitializePlugin %(AdditionalOptions)</AdditionalOptions>
-      <AdditionalDependencies>Foundation.lib;OpenMaya.lib;OpenMayaAnim.lib;OpenMayaRender.lib;OpenMayaUI.lib;cg.lib;cgGL.lib;OpenGL32.lib;winmm.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>Foundation.lib;OpenMaya.lib;OpenMayaAnim.lib;OpenMayaRender.lib;OpenMayaUI.lib;OpenMayaFX.lib;cg.lib;cgGL.lib;OpenGL32.lib;winmm.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
OpenMayaFx was to get gravityfield for physic maya object.
Now i use Bullet so no more needs.
i remove it

http://download.autodesk.com/us/maya/2010help/API/class_m_fn_gravity_field.html